eZ Platform/Admin UI/REQUIREMENTS.md

```diff
@@ -0,0 +1,45 @@
+# v2 Hybrid Prototype Admin UI Requirements
+## https://github.com/ezsystems/specs/issues/4
+
+# Objective
+"As an eZ Engineer, I want...
+
+To create a functioning, experimental combination of Symfony3, eZ Platform, and
+a mix of existing elements (in YUI) and rewritten elements (in a new framework),
+for the purpose of research and development (prototype), so that I can get a
+better picture of how such a hybrid approach may work."
+
+Underlying main goals are:
+- by adopting Symfony 3, we will provide developers, and especially the one who are experienced with eZ Publish or Symfony, substantially faster and easier way to develop, extend and customize eZ Platform.
+- by insuring an hybrid architecture that reuses existing elements relying on YUI, we will speed up availability of 2.x by not having to redevelop all features. Candidates for reuse are UDW, Sub-items view, Content view. If this is not possible, expectations and plans for 2.x will have to be be re-adjusted.
+
+# Definitions
+- "Hybrid Approach": A Multi-Page App (MPA) approach, replacing current SPA
+  - Uses Symfony routing and insure we can have application URL that can use content ids, location ids, content fields (name)
+  - Allows extending Admin UI via Symfony/Twig
```

Comment:
A first list of extension points that we should focus on would be nice. I see at very least:
* Custom link in Navigation Hub + what we used to call "Custom module" in legacy rendered server side
* Field view and Field Edit View for custom field types

but I'm sure there are several others we should consider important as well.